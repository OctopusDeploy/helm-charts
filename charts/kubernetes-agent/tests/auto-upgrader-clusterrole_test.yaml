suite: "pod permissions"
templates:
- templates/auto-upgrader-clusterrole.yaml
tests:
- it: "should match snapshot"
  asserts:
  - matchSnapshot: {}
