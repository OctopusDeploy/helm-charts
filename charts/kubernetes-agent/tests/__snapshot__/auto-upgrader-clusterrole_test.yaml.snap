should match snapshot:
  1: |
    apiVersion: rbac.authorization.k8s.io/v1
    kind: ClusterRole
    metadata:
      name: octopus-agent-auto-upgrader-RELEASE-NAME-role
    rules:
      - apiGroups:
          - '*'
        resources:
          - '*'
        verbs:
          - '*'
